recipes/rhapsody/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "0.2.1" %}
+{% set name = "rhapsody" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/r/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d73e0af9f57fdb7e48bd7bab4f946fbeb0240f6d44cb47e0257a277df7051beb
+
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+  entry_points:
+    - qiime2.plugins = rhapsody.q2.plugin_setup:plugin
+
+requirements:
+  host:
+    - python <3.7
```

Does this really support all versions except Python 3.7? The [metadata](https://github.com/biocore/rhapsody/blob/master/setup.py#L21-L22) implies this should be:
```suggestion
    - python >=3
```
(If this changes it should also change in `run`)